### ISSUE TITLE
Question - Material UI example

### ISSUE THREAD

**Issue by jtrunick** [cleandart/react-dart#109]
Is it possible to use a component from the Material UI components? If so, is there an example?

(didn't know where to post this question)

**Comment by greglittlefield-wf** [comment]
Yes, it is possible to use JavaScript React components, though it's not very straightforward at this time... We can use this issue to track the documentation/support around that.

There was some code written to use JS components in an older version of react-dart, but that needs to be updated. I'll see if I can get some code posted early this week.

**Comment by greglittlefield-wf** [comment]
Sorry for the delay! I blame the holidays 😄.

Got together a PR to add support and an example for using JS components with react-dart: https://github.com/cleandart/react-dart/pull/112. A couple tests to add still, but it should be about done

Also, I went ahead and ported one of the Material-UI examples to use react-dart! You can see that here: https://github.com/greglittlefield-wf/material-ui-webpack-example 

Let me know if you have any questions about that code or any feedback regarding this work!

**Comment by jestan** [comment]
Sorry to comment on old issue.
Is there any plan to support this feature?
It will be very useful to reuse existing reactjs components in react-dart/over_react

**Comment by aaronlademann-wf** [comment]
@greglittlefield-wf ^

**Comment by aaronlademann-wf** [comment]
@jestan @GiovanniL19 @Sach97 @ayushin @jtrunick I believe this will be supported / documented in the upcoming react-dart version 5.0.0 _(support for ReactJS 16)_ via #160.
